src/modules/integrator-guide/pages/corteza-studio/index.adoc

```diff
@@ -0,0 +1,139 @@
+include::ROOT:partial$variables.adoc[]
+
+= {PRODUCT_NAME} Studio
+
+:description: {PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences.
+{PRODUCT_NAME} Studio allows users to customize {PRODUCT_NAME} web applications styling according to their preferences. 
+
+[IMPORTANT]
+====
+For {PRODUCT_NAME} Studio, to function properly, you need to <<installing-dart-sass, Install dart-sass binaries>> in your system and 
+`HTTP_WEBAPP_ENABLED` enviroment variable set to `true`.
```

Comment:
Same here.
When I add comments I often don't duplicate them elsewhere (or I just miss something) so please re-check the entire document and address all occurrences